build-tools/installers/create-installers.targets

```diff
@@ -165,6 +165,7 @@
     <_MSBuildFiles Include="$(MicrosoftAndroidSdkOutDir)libZipSharp.dll" />
     <_MSBuildFiles Include="@(_LocalizationLanguages->'$(MicrosoftAndroidSdkOutDir)%(Identity)\libZipSharp.resources.dll')" />
     <_MSBuildFiles Include="$(MicrosoftAndroidSdkOutDir)libZipSharp.pdb" />
+    <_MSBuildFiles Include="$(MicrosoftAndroidSdkOutDir)MavenNet.dll" />
```

Comment:
Do "we" own this? Does it have a `.pdb`?

When we insert in VS, it will complain if we don't have a `.pdb` for it.

Reply:
MavenNet is Redth's tool

Reply:
This is already underway.  :)

https://github.com/Redth/MavenNet/pull/13